src/codegen/compile.ml

```diff
@@ -5299,7 +5331,7 @@ module MakeSerialization (Strm : Stream) = struct
       | Prim Blob ->
         add_typ Type.(Array (Prim Nat8))
       | Prim Region ->
-        add_typ (Type.Prim Region)
+        add_sleb128 idl_alias; add_idx t
```

Comment:
This is a bit fishy: `add_typ` isn’t used for primitive type before (see below), but I think you wanted to make `Region` primitive?

But it needs the alias type tag _and_ something to put int. So far, these are always different types `Array (Mut t)` vs. `Array t`; `Mut t` vs `t`. Here you want to use `Prim Region` for both the aliased thing and the thing inside, I guess. So maybe fine.